ui/app/src/main/java/com/example/fuel/repository/UserRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.fuel.repository
+
+import com.example.fuel.api.RetrofitInstance
+import com.example.fuel.model.User
+import com.example.fuel.viewmodel.UserViewModel
+import retrofit2.Response
+import retrofit2.http.Body
+
+class UserRepository {
+
+    suspend fun postLogin(user: User): Response<User> {

```

Comment:
Body of response does not contains same fields as body of request. You will receive something like this:
![image](https://user-images.githubusercontent.com/61120635/201300682-fd7c60a5-6375-47a4-8acd-1e1e484aa391.png)

So it should be mapped into another model.